Gopkg.toml

```diff
@@ -27,11 +27,11 @@
 
 [[constraint]]
   name = "github.com/cloudtrust/common-service"
-  version = "v1.0-rc6"
+  version = "=v1.0-rc6"
```

Comment:
Why = ?
Is it an error or a feature I don't know ?

Reply:
= in toml means exactly this version. If there is no =, it only means >=.
Here, I think we should remove the =

Reply:
I added the = as the error handling changes was causing otherwise problems;

Reply:
DONE